docs/developers/library_examples.md

```diff
@@ -0,0 +1,613 @@
+# Library Examples
+
+## C
+
+Coming Soon ...
```

Comment:
still true?

Reply:
Maybe. I'm still thinking about this one.

Doxygen may be enough for the C APIs. However, it does seem like a good idea to provide examples around common operations. Some users will be afraid to use C++ due to complexity. For that reason, C may be a better fit. Here are a few things I feel would be good to have here for everyone, including the core dev team:
- Connecting to iRODS
- Iterating over a collection
- Using GenQuery

The Library Examples section can serve as a quick reference for various things instead of requiring devs to look through Doxygen documentation.

Reply:
reasonable list.   i think that makes sense.